### ISSUE TITLE
FDroid Build

### ISSUE THREAD

**Issue by Dav-Git** [jheubuch/traewelling-android#184]
**Is your feature request related to a problem? Please describe.**
I cannot find Träwelldroid on the FDroid store. I would however like to not use the google play store while still being able to install Träwelldroid.

**Describe the solution you'd like**
A Träwelldroid build to be published on FDroid.

**Describe alternatives you've considered**
Sideloading by building the app myself. This is very cumbersome to do regularly though.

**Additional context**
I did some initial research (see [here](https://f-droid.org/docs/FAQ_-_App_Developers/)) but I'm really out of the loop on android development. Hopefully this page can help you get started :)

**Comment by jheubuch** [comment]
Hi,
this is already discussed in #36. 
TL;DR
For the moment it isn't planned (but wished for the future) to release Träwelldroid on F-Droid. You have the opportunity to download the apk-Files from the Releases section until I privately have the chance to investigate into this topic. (But I can't tell whenever this will be)
